crates/web-sys/webidls/enabled/CSSStyleSheet.webidl

```diff
@@ -24,4 +26,8 @@ interface CSSStyleSheet : StyleSheet {
   unsigned long insertRule(DOMString rule, optional unsigned long index = 0);
   [Throws, NeedsSubjectPrincipal]
   undefined deleteRule(unsigned long index);
+  [NewObject]
+  Promise<CSSStyleSheet> replace(DOMString text);
+  [Throws]
+  undefined replaceSync(DOMString text);
```

Comment:
```suggestion
  [NewObject]
  Promise<CSSStyleSheet> replace(USVString text);
  [Throws]
  undefined replaceSync(USVString text);
```
According to the spec: https://drafts.csswg.org/cssom/#ref-for-dom-cssstylesheet-replace.